interfaces/cython/cantera/ctml_writer.py

```diff
@@ -1319,9 +1319,11 @@ def build(self, p):
             self._kf = []
 
         if self._type == 'edge' or self._type == 'surface':
-            if self._beta > 0:
+            if self._beta is not None:
                 electro = kfnode.addChild('electrochem')
                 electro['beta'] = repr(self._beta)
+            else:
+                self._beta = 0.0
```

Comment:
I don't think it's necessary to set a value for `_beta` here. This would just have the awkward effect of having the awkward effect of having the result of `build` change if it was called a second time (not that this actually happens in practice).

Reply:
Okay, thanks @speth. Just went in and corrected that.